src/infra/brick.go

```diff
@@ -43,7 +48,10 @@ type Brick struct {
 	// A pointer to a module
 	Module *Module
 	// Pointer to the bricks it depends on
-	Dependencies []Dependency
+	DirectPrevious Bricks
+
+	// Data (and their represenation) from other bricks output that brick need to plan,lay,remove,output
+	Input []Input
```

Comment:
Plutôt pluriel ici du coup, non?
```suggestion
	Inputs []Input
```